### PR TITLE
Blog post on recorded talks this year

### PR DESCRIPTION
**Pull request by amirmc**
TODO

- [x] Come up with a better title :)
- [x] Update the videos page with the same links
- [ ] Update the /links data

I'd appreciate any suggestions.

### CODE REVIEW

**Comment by avsm** [comment]
Added a couple more places on the site that could use the content

File: tmpl/blog/recent-talks-oct2015.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+one of your videos, please do [get in touch][mail]!
+
+#### Videos of recent talks
+
+**[Anil Madhavapeddy][anil] at Esper Technologies - May 2015**
+<div class="flex-video">
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/bC7rTUEZfmI" frameborder="0" allowfullscreen></iframe>
+</div>
+
+**[Russell Pavlicek][russ]** at SouthEast LinuxFest - June 2015
```
Comment (dsheets) [inline tmpl/blog/recent-talks-oct2015.md]: Should double star whole line?

**Comment by dsheets** [comment]
How about the titles of the talks on the same page?

**Comment by amirmc** [comment]
Thanks for the comments.  I've added them to the videos page but adding to links will mean editing a lot of metadata.  I'd like to push this live as soon as CI is green so I'll leave /links for another time.

**Comment by amirmc** [comment]
Actually, I'm merging this now as it'll have to build on `master` again anyway.  It's builds just fine locally but I'll keep an eye on it.

**Pull request merged**